.gitignore

```diff
@@ -33,14 +32,10 @@ prerelease.txt
 # Build directory
 /build*
 emscripten_build/
-docs/_build
-docs/_static/robots.txt
-__pycache__
-docs/utils/*.pyc
-/deps/downloads/
-deps/install
-deps/cache
-cmake-build-*/
+/docs/_build
+/docs/_static/robots.txt
+/docs/utils/
```

Comment:
I think this one should not be ignored:
```suggestion
```

Checking history, [it used to contain the lexer](https://github.com/ethereum/solidity/tree/v0.4.26/docs/utils), which is now a standalone library: [pygments-lexer-solidity](https://gitlab.com/veox/pygments-lexer-solidity).